modules/@apostrophecms/widget-type/index.js

```diff
@@ -97,7 +97,10 @@ const _ = require('lodash');
 module.exports = {
   cascades: [ 'fields' ],
   options: {
-    neverLoadSelf: true
+    neverLoadSelf: true,
+    initialModal: true,
+    placeholder: false,
+    placeholderClass: 'apos-placeholder'
```

Comment:
Add these by default, so a developer is aware they exist just by checking the `widget-type` module options